recipes/qdldl-python/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "0.1.3" %}
+
+package:
+  name: qdldl-python
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/q/qdldl/qdldl-{{ version }}.tar.gz
+  sha256: 2e98bfcc43de55225b82f5895f8266e79a305b174490c60bc559c1538f1be920
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - numpy
```

Comment:
Do you link against numpy? If so please follow the instructions here: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
oh nm you are